staging/src/k8s.io/api/admissionregistration/v1/generated.pb.go

```diff
@@ -342,13 +342,13 @@ func init() {
 }
 
 var fileDescriptor_aaac5994f79683e8 = []byte{
-	// 1104 bytes of a gzipped FileDescriptorProto
+	// 1102 bytes of a gzipped FileDescriptorProto
```

Comment:
looks like the same changes we saw in this PR https://github.com/kubernetes/kubernetes/pull/98363, since we did not cherry-pick that to the release branches this PR need to update those

Reply:
@justaugustus @dims - is there a way to get a builder image with the go version we need that does not include the protoc bump? I would not expect us to have to change generators to pick up a minor go version

Reply:
@liggitt the protoc bump was a one time thing when we updated kube-cross to include arm64 (and the specific version of protoc we had earlier did not have a arm64 variant, so we picked the oldest protoc which was arm64 enabled)

Reply:
is there a way to get a 1.15.8 builder without that so we can minimize touches to API files in release branches?

Reply:
If that was included because of the arm64 variant, we might need to build another kube-cross without the variant I guess

Reply:
Pr to create the new variant: https://github.com/kubernetes/release/pull/1915

Reply:
ready @liggitt @dims @justaugustus